File: java-sdk/com/semantria/objects/output/CollAnalyticData.java
package com.semantria.objects.output;

import java.util.LinkedList;
import java.util.List;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlElementWrapper;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name="collection")
public class CollAnalyticData 
{
	private String id;
	private String config_id;
	private TaskStatus status;
	private List<Facet> facets;
    private List<CollTopic> topics;
    private List<CollTheme> themes;
    private List<CollEntity> entities;
	
	@XmlElement(name="id")
	public String getId() { return id; }
	@XmlElement(name="config_id")
	public String getConfigId() { return config_id; }
	@XmlElement(name="status")
	public TaskStatus getStatus() { return status; }
	@XmlElementWrapper(name="facets")
	@XmlElement(name="facet")
	public List<Facet> getFacets() { return facets; }
    @XmlElementWrapper(name="entities")
    @XmlElement(name="entity")
    public List<CollEntity> getEntities() { return entities; }
    @XmlElementWrapper(name="topics")
    @XmlElement(name="topic")
    public List<CollTopic> getTopics() { return topics; }
    @XmlElementWrapper(name="themes")
    @XmlElement(name="theme")
    public List<CollTheme> getThemes() { return themes; }
	
	public void setId(String cid) { id = cid; }
	public void setConfigId(String config) { config_id = config; }
	public void setStatus(TaskStatus cstatus) { status = cstatus; }
	public void setFacets(List<Facet> list) { facets = list; }
    public void setEntities(List<CollEntity> sentities) { entities = sentities; }
    public void setTopics(List<CollTopic> atopics) { topics = atopics; }
    public void setThemes(List<CollTheme> athemes) { themes = athemes; }
}

File: dotNet/Semantria/Mapping/Output/CollAnalyticData.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
	[DataContract(Name = "collection")]
	[XmlRootAttribute("collection", Namespace = "")]
	public class CollAnalyticData
	{
		[DataMember(Name = "id")]
		[XmlElementAttribute("id")]
		public string Id { get; set; }

        [DataMember(Name = "tag")]
        [XmlElementAttribute("tag")]
        public string Tag { get; set; }

		[DataMember(Name = "config_id")]
		[XmlElementAttribute("config_id")]
		public string ConfigId { get; set; }

		[DataMember(Name = "status")]
		private string _status;
		[XmlElementAttribute("status")]
		public TaskStatus Status
		{
			get
			{
				if (!String.IsNullOrEmpty(_status))
				{
					return (TaskStatus)Enum.Parse(typeof(TaskStatus), _status, true);
				}
				else
				{
					return TaskStatus.UNDEFINED;
				}
			}
			set { _status = value.ToString(); }
		}

		[DataMember(Name = "facets")]
		[XmlArrayAttribute("facets")]
		[XmlArrayItemAttribute("facet", typeof(Facet))]
		public List<Facet> Facets { get; set; }

		[DataMember(Name = "themes")]
		[XmlArrayAttribute("themes")]
		[XmlArrayItemAttribute("theme", typeof(CollTheme))]
		public List<CollTheme> Themes { get; set; }

		[DataMember(Name = "topics")]
		[XmlArrayAttribute("topics")]
		[XmlArrayItemAttribute("topic", typeof(CollTopic))]
		public List<CollTopic> Topics { get; set; }

		[DataMember(Name = "entities")]
		[XmlArrayAttribute("entities")]
		[XmlArrayItemAttribute("entity", typeof(CollEntity))]
		public List<CollEntity> Entities { get; set; }
	}
}

namespace Semantria.Com.Mapping.Output.Stub
{
	[DataContract(Name = "collections")]
	[XmlRootAttribute("collections", Namespace = "")]
	public class CollAnalyticsData
	{
		[DataMember(Name = "collection")]
		[XmlElementAttribute("collection")]
		public List<CollAnalyticData> Data { get; set; }
	}
}
